adal/src/androidTest/java/com/microsoft/aad/adal/OauthTests.java

```diff
@@ -51,30 +55,34 @@
 import java.util.UUID;
 import java.util.concurrent.CountDownLatch;
 
+import static junit.framework.TestCase.fail;
```

Comment:
same comment as before for unused imports, i may have missed some places, please scrub the entire PR for unused imports.